lms/static/js/search/course/main.js

```diff
@@ -0,0 +1,22 @@
+RequireJS.require([
+    'jquery',
+    'backbone',
+    'js/search/course/search_app',
+    'js/search/base/routers/search_router',
+    'js/search/course/views/search_form',
+    'js/search/base/collections/search_collection',
+    'js/search/course/views/search_results_view'
+], function ($, Backbone, SearchApp, SearchRouter, CourseSearchForm, SearchCollection, CourseSearchResultsView) {
+    'use strict';
+
+    var courseId = $('#courseware-search-results').attr('data-course-id');
```

Comment:
Personally, I like `.data('course-id')` instead of `.attr('data-course-id')` - I like the use of `data` as an accessor to data attributes, I think it reads better.